File: src/main/java/it/contrader/dto/DeviceDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class DeviceDTO {

	private Integer iddevice;

//	private int idclient;
	private ClientDTO client;
	
	private String mac;
	
	private String devtype;

	private String position;
	
}

File: src/dto/devicedto.ts
import { ClientDTO } from './clientdto';

/**
 * Classe DTO di User. DEVE essere uguale (stesso nome classe, stessi attributi e stessi nomi) a
 * quello nel backend. 
 * 
 * @author PattyGè
 */
export class DeviceDTO {

    id: number;

	clientDTO: ClientDTO;
	
	mac: string;

	devtype: string;

	position: string;
    
}